docs/source/_static/depthai_env_setup.sh

```diff
@@ -177,3 +199,127 @@ else
+BASHRC="$HOME/.bashrc"
+ADD_DEPTHAI_TO_PATH='export PATH=$PATH'":$WORKING_DIR"
+ADD_LUXONIS_DEPTHAI_HOME='export LUXONIS_DEPTHAI_HOME='"$WORKING_DIR"
+
+# add to .bashrc only if it is not in there already
+write_in_file "$COMMENT" "$BASHRC"
+write_in_file "$ADD_DEPTHAI_TO_PATH" "$BASHRC"
+write_in_file "$ADD_LUXONIS_DEPTHAI_HOME" "$BASHRC"
+
+
+if [[ $(uname -s) == "Darwin" ]]; then
+  echo _____________________________
+  echo "Calling macOS_installer.sh"
+  echo _____________________________
+  echo "Running macOS installer."
+  echo "Upgrading brew."
+  brew upgrade
+
+  # clone depthai form git
+  if [ -d "$WORKING_DIR" ]; then
+     echo "Demo app already downloaded. Checking out main and updating."
+  else
+     echo "Downloading demo app."
+     git clone https://github.com/luxonis/depthai.git "$WORKING_DIR"
+  fi
+  cd "$WORKING_DIR"
+  git fetch
+  git checkout demo_app_installation_v2
+  git pull
+
+  # install python 3.10 and python dependencies
+  brew update
+  echo "installing python 3.10"
+  brew install python@3.10
+
+  # pip does not have pyqt5 for arm
+  if [[ $(uname -m) == 'arm64' ]]; then
+    echo "Installing pyqt5 with homebrew."
+    brew install pyqt@5
+  fi
+
+  # create python virtual environment
+  echo "Creating python virtual environment in $WORKING_DIR/venv"
+
+  python3.10 -m venv "$WORKING_DIR/venv"
+  # activate environment
+  source "$WORKING_DIR/venv/bin/activate"
+  pip install --upgrade pip
+
+  # install launcher dependencies
+  # only on mac silicon point PYTHONPATH to pyqt5 installation via homebrew, otherwise install pyqth5 with pip
+  if [[ $(uname -m) == 'arm64' ]]; then
+    #if [[ $(uname -s) == "Darwin" ]]; then
+    if [[ ":$PYTHONPATH:" == *":/opt/homebrew/lib/python3.10/site-packages:"* ]]; then
+      echo "/opt/homebrew/lib/python3.10/site-packages already in PYTHONPATH"
+    else
+      export PYTHONPATH=/opt/homebrew/lib/python3.10/site-packages:$PYTHONPATH
+      echo "/opt/homebrew/lib/python3.10/site-packages added to PYTHONPATH"
+    fi
+    #fi
+  else
+    pip install pyqt5
+  fi
+
+  pip install packaging
+
+elif [[ $(uname -s) == "Linux" ]]; then
+  echo _____________________________
+  echo "Calling linux_installer.sh"
+  echo _____________________________
+  echo $'\nRunning Linux installer.'
+  echo "Updating sudo-apt."
+  echo "$WORKING_DIR"
+
+  sudo apt-get update
+
+  # clone depthai form git
+
+  if [ -d "$WORKING_DIR" ]; then
+     echo "Demo app already downloaded. Checking out main and updating."
+
+  else
+     echo "Downloading demo app."
+     git clone https://github.com/luxonis/depthai.git "$WORKING_DIR"
+  fi
+
+  cd "$WORKING_DIR"
+  git fetch
+  git checkout demo_app_installation_v2
+  git pull
+
+  # install python 3.10
+  echo "installing python 3.10"
+
+  sudo yes "" | sudo add-apt-repository ppa:deadsnakes/ppa
+  sudo apt -y install python3.10
+  sudo apt -y install python3.10-venv
```

Comment:
Ok, I agree, but when I look at install_dependencies.sh:

readonly linux_pkgs=(
    python3
    python3-pip
    udev
    cmake
    git
    python3-numpy
)
That would mean we should delete python3, python3-pi and python3-numpy, because otherwise we will install it no matter what on Linux systems and that is against your logic. What do you think? @themarpe

Reply:
That installs the default python3 available for the system - not optional if someone would like something completely custom, but 99% aligned with normal usage.

Whereas installing custom version can lead to conflicts then with multiple python version, packages, etc...
So preferably we just rely on `python3` as is, whichever version it is (as long as it is a supported one)

Reply:
Also edited the above comment - I meant optionally selectable.

But relying on python3 for now will do